youtube2zim/scraper.py

```diff
@@ -550,24 +572,71 @@ def get_slot():
 
         return overall_succeeded, overall_failed
 
-    @staticmethod
-    def download_video_files_batch(options, videos_ids):
+    def download_s3_cache(self, s3_key, local_video_path):
```

Comment:
rename to `download_from_cache` and use `""" whether it successfully downloaded from cache """`